docs/_main/getting_started.md

```diff
@@ -22,23 +22,19 @@ status. Before we can call any methods, we need to configure our new app with th
 ## Getting a token to use the Web API
 
 Navigate to "OAuth & Permissions" and scroll down to the section for scopes. Slack describes the various permissions
-your app could obtain from an installing user as **scopes**. There are [over 60 scopes](https://api.slack.com/scopes)!
+your app could obtain from an installing bot as **scopes**. There are [over 80 scopes](https://api.slack.com/scopes)!
 Some are broad and authorize your app to access lots of data, while others are very specific and let your app touch just
 a tiny sliver. Your users (and their IT admins) will have opinions about which data your app should access, and only
 agree to install the app if the data permissions seem reasonable, so we recommend finding the scope(s) with the least
 amount of privilege for your app's needs. In this guide we will use the Web API to post a message. The scope required
-for this is called `chat:write:user`. Use the dropdown or start typing its name to select and add the scope, then click
+for this is called `chat:write`. Use the dropdown or start typing its name to select and add the scope, then click
 "Save Changes".
 
-Now our app has described which scope it desires in the workspace, but nobody has authorized those scopes for your
-workspace yet. Scroll up and click "Install App". You'll be taken to your app installation page. This page is asking you
-for permission to install the app in your development workspace with specific capabilities. That's right, the
-development workspace is like every other workspace -- apps must be authorized by a user each time it asks for more
-permissions.
+Now our app has described which scope it desires in the workspace, but we haven't added it to your workspace yet. To do so, we first need to create a bot user. Scroll down to Bot Users and click "Add a Bot User". Add a display name and username for your bot. Next, scroll up to Install App and click "Install App to Workspace". You'll be taken to the app installation page. This page is where you grant the bot user permission to install the app in your development workspace with specific capabilities.
```

Comment:
"Next, scroll up to Install App and click "Install App to Workspace"." since the user will automatically be on the OAuth & Permissions page they can just scroll to the top of the page and see the "Install App to Workspace" button. I'd consider changing it to say "To install your app, scroll up to the top of the page and click the green **Install App to Workspace** button".